src/components/AnonymousChat/child/ChannelImage.tsx

```diff
@@ -140,6 +141,7 @@ const ChannelImage = ({
           imageStyle={styles.image}
         />
       );
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The closing brace for the if statement has been added. This is necessary due to the addition of the opening brace in the previous comment.
```diff
+    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->